src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java

```diff
@@ -281,7 +456,7 @@ private void addTag(String text, int line) {
     /**
      * A Tag holds a suppression comment and its location.
      */
-    private static final class Tag {
+    private static class Tag {
```

Comment:
Still a continuation of https://github.com/checkstyle/checkstyle/pull/6877#discussion_r300837966 ,
Why is this changing?

Reply:
Sorry, my bad. Now fixed.